src/codeu/chat/util/TokenizerTest.java

```diff
@@ -0,0 +1,26 @@
+package codeu.chat.util;
+ 
+import java.io.IOException;
+import static org.junit.Assert.*;
+import org.junit.Test;
+import codeu.chat.util.Tokenizer;
+ 
+public final class TokenizerTest {
+   @Test
+   public void testWithQuotes() throws IOException {
+      final Tokenizer tokenizer = new Tokenizer("hello world how are you");
+      assertEquals(tokenizer.next(), "hello");
+      assertEquals(tokenizer.next(), "world");
+      assertEquals(tokenizer.next(), "how");
+      assertEquals(tokenizer.next(), "are");
+      assertEquals(tokenizer.next(), "you");
+      assertEquals(tokenizer.next(), null);
+   }
```

Comment:
One of those extra blank lines from somewhere else can move here to separate these two tests. ;-)

@nvaa36 @iZake